doc/gvmd.8.xml

```diff
@@ -382,6 +382,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
            as used in GMP.</p>
       </optdesc>
     </option>
+    <option>
+      <p><opt>--scanner-connection-retry=<arg>NUMBER</arg></opt></p>
+      <optdesc>
+        <p>During a running task, number of auto retry on scanner connection
```

Comment:
I think a better way to phrase this would be
"Number of auto retries if scanner connection is lost in a running task."